astroquery/mast/tests/test_mast_remote.py

```diff
@@ -1088,3 +1088,71 @@ def test_zcut_get_cutouts(self):
         assert isinstance(cutout_list, list)
         assert len(cutout_list) == 1
         assert isinstance(cutout_list[0], fits.HDUList)
+
+    ###################
+    # HapcutClass tests #
+    ###################
+
+    def test_hapcut_download_cutouts(self, tmpdir):
+        
+        # Test 1: Simple API call with expected results
+        coord = SkyCoord(351.347812, 28.497808, unit="deg")
+
+        cutout_table = mast.Hapcut.download_cutouts(coordinates=coord, size=5, path=str(tmpdir))
```

Comment:
nitpick: ideally tests like these can be parametrized, so they would be independent and would not require repeated code

Reply:
Agreed, I think parametrization of some of these tests will be straight forward after consolidating all the classes into 1. One problem I see us running into is that some of the example inputs we use (mainly for the `coordinates` arg) only work in one tool and not the other, e.g. and RA/Dec of `102.7, 70.50` will work for TESSCut and not HAPCut. So I'll need to find a set of parameters that will work for all 3. But that's a bridge we can cross when we get to it.

Reply:
oh, I didn't mean to parametrize between classes, but for `size` within each of these tests, with a `pytest.mark.parametrize` (as practically that's the only thing changing within the test)